applications/nrf5340_audio/src/audio/sw_codec_select.c

```diff
@@ -112,8 +112,8 @@ int sw_codec_decode(uint8_t const *const encoded_data, size_t encoded_size, bool
 	char pcm_data_mono[PCM_NUM_BYTES_MONO] = {0};
 	static char pcm_data_stereo[PCM_NUM_BYTES_STEREO];
 
-	size_t pcm_size_stereo = 0;
-	size_t pcm_size_session = 0;
+	uint16_t pcm_size_stereo = 0;
+	uint16_t pcm_size_session = 0;
```

Comment:
Should do the necessary changes for SW_CODEC_MONO as well

Reply:
Adding (size_t *) and removing (uint16_t *)

Reply:
Right you are, forgot to remove the pointer cast from mono, so it was casting a (uint16_t *) to (uint16_t *) :P Fixed

Reply:
The (size_t *) was already added, to zero_pad